packages/apps/transfer/src/pages/faucet/existing/index.tsx

```diff
@@ -123,7 +123,11 @@ const ExistingAccountFaucetPage: FC = () => {
               leftIcon: SystemIcon.KIcon,
             }}
           />
-          <ChainSelect onChange={onChainSelectChange} value={chainID} />
+          <ChainSelect
+            onChange={onChainSelectChange}
+            value={chainID}
+            ariaLabel="Account name"
+          />
```

Comment:
Nice, I saw this issue in `main` too. A bit odd to put these changes in a single PR, but fine by me.

Reply:
Ah yeah, happened to spot it and it seems minor so I figured I'll fix it right away.